server/index-node/src/schema.graphql

```diff
@@ -95,3 +96,10 @@ enum Feature {
   fullTextSearch,
   ipfsOnEthereumContracts,
 }
+
+type SubgraphVersion {
+  """
+  Version number in SemVer format
+  """
+  version: String!
+}
```

Comment:
Why an object instead of a list? It's extendable and harder to break backwards-compatibility.